src/webui/frontend/src/actions/rules.ts

```diff
@@ -35,7 +35,9 @@ export const loadSnowAlertRules = () => async (dispatch: Dispatch, getState: Get
       const response = await api.loadSnowAlertRules();
       dispatch(LoadRulesActions.loadSnowAlertRulesSuccess(response.rules));
     } catch (error) {
-      dispatch(LoadRulesActions.loadSnowAlertRulesFailure(error.message));
+      if (error instanceof Error) {
```

Comment:
does it work and might it make more sense to type hint `: any` here, as well? has this been tested and will this ever not exec a dispatch?

Reply:
It will not exec a dispatch in case something odd, which is not an error is thrown, which may make `error.message` result in an `undefined`.  Sorry yes, `: any` seems like a better choice here, since it would accommodate cases like that too.